starter-code/routes/auth.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+const User = require('../models/user')
+
+/* SINGUP */
+router.get('/signup', (req, res, next) => {
+    res.render('auth/signup')
+});
+
+router.post('/signup', (req, res, next) => {
+    const { username, password } = req.body
+    if (username == "" || password == "") {
+        res.send('Username and password can\'t be empty')
+    } else {
+        bcrypt.hash(password, 10)
+        .then(hash => {
+            return User.create({
+                username: username,
+                password: hash
+            })
+            .then(user => {
+                res.send('WELCOME!!!')
```

Comment:
You can also render a view page and send the username to the page like res.render('profile', {username: user.username})